### PR TITLE
k8s: Update to final v1.29.0

### PR DESCRIPTION
**Pull request by christarazi**
Signed-off-by: Chris Tarazi <chris@isovalent.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -111,10 +111,10 @@ require (
...
 	gopkg.in/ini.v1 v1.67.0
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.29.0-rc.1
+	k8s.io/api v0.29.0
 	k8s.io/apiextensions-apiserver v0.29.0-rc.1
-	k8s.io/apimachinery v0.29.0-rc.1
-	k8s.io/client-go v0.29.0-rc.1
+	k8s.io/apimachinery v0.29.0
+	k8s.io/client-go v0.29.0
 	k8s.io/code-generator v0.29.0-rc.1
```
Comment (sayboras) [inline go.mod]: do we need to update code-generator as well ?

File: go.mod [hunk 2]
```diff
@@ -111,10 +111,10 @@ require (
 	google.golang.org/protobuf v1.31.0
 	gopkg.in/ini.v1 v1.67.0
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.29.0-rc.1
+	k8s.io/api v0.29.0
 	k8s.io/apiextensions-apiserver v0.29.0-rc.1
```
Comment (sayboras) [inline go.mod]: do we need to update this one as well?

**Review by tklauser — APPROVED** [review]

**Pull request merged**